src/SFA.DAS.EmployerFinance.Jobs/ScheduledJobs/ProcessLevyDeclarationsJob.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using NServiceBus;
+using SFA.DAS.EmployerFinance.Application.Commands.ProcessLevyDeclarations;
+using SFA.DAS.EmployerFinance.Services;
+
+namespace SFA.DAS.EmployerFinance.Jobs.ScheduledJobs
+{
+    public class ProcessLevyDeclarationsJob
+    {
+        private readonly IMessageSession _messageSession;
+        private readonly IDateTimeService _dateTimeService;
+
+        public ProcessLevyDeclarationsJob(IMessageSession messageSession, IDateTimeService dateTimeService)
+        {
+            _messageSession = messageSession;
+            _dateTimeService = dateTimeService;
+        }
+
+        public Task Run([TimerTrigger("0 0 15 20 * *")] TimerInfo timer, ILogger logger)
+        {
+            var now = _dateTimeService.UtcNow;
+            var today = now.Date;
+            var month = new DateTime(today.Year, today.Month, 1, 0, 0, 0, 0, DateTimeKind.Utc);
+            var payrollPeriod = month.AddMonths(-1);
+            var command = new ProcessLevyDeclarationsCommand(payrollPeriod);
+            var task = _messageSession.Send(command);
```

Comment:
This actually sends the message via the bus' transport (it's not handled in the `SFA.DAS.EmployerFinance.Jobs` process).

Reply:
Ah yes I see now.

I was confused because the sent command `ProcessLevyDeclarationsCommand` is a Mediatr `IRequest` and lives with the rest of the Mediatr commands in Application\Commands, but it's actually both a Mediatr command and an NServiceBus command handled by `ProcessLevyDeclarationsCommandHandler` in MessageHandlers, which is the context that the command handlers run in.